backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/document/DocTypeSequenceMap.java

```diff
@@ -72,12 +73,27 @@ public String toString()
 				.toString();
 	}
 
-	public DocSequenceId getDocNoSequenceId(final ClientId adClientId, final OrgId adOrgId)
+	public DocSequenceId getDocNoSequenceId(final ClientId adClientId, final OrgId adOrgId, final CountryId countryId)
 	{
 		if (!docTypeSequences.isEmpty())
 		{
-			final ArrayKey key = mkKey(adClientId, adOrgId);
-			final DocTypeSequence docTypeSequence = docTypeSequences.get(key);
+			ArrayKey key = mkKey(adClientId, adOrgId, countryId);
```

Comment:
here we have an "implicit" (ofc it's explicit, but only if you look at the code) match-ordering.
For the user it needs some detective-work to deduce it.
Friendly reminder: i asked you to have a `SeqNo` column. This is why :-)